ops/lib.rs

```diff
@@ -209,7 +208,7 @@ fn codegen_v8_async(
   let rust_i0 = special_args.len();
   let args_head = special_args.into_iter().collect::<TokenStream2>();
 
-  let (arg_decls, args_tail, argc) = codegen_args(core, f, rust_i0, 1, true);
+  let (arg_decls, args_tail, _) = codegen_args(core, f, rust_i0, 1, true);
```

Comment:
Intentially don't want to remove the actual arg count computation. Just to save a few mins if we decide to add back the async op codegen in some other way.

Reply:
Good thinking!